README.md

```diff
@@ -38,19 +53,52 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 You can edit the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
 
+
 ## Docker
 
 1. [Install Docker](https://docs.docker.com/get-docker/) on your machine.
 2. Build the image and run the docker compose setup: `docker compose -f docker-compose-development.yml up`.
 
-You can view your images created with `docker images`.
+You can view the images you create with `docker images`.
 
 Open [http://localhost](http://localhost) with your browser to see the result. You can visit [http://localhost:8080](http://localhost:8080) to see the reverse proxy setup before the main application.<!-- markdown-link-check-disable-line -->
 
 The `docker-compose.yml` setup contains a production-ready setup using a reverse proxy.
 
 Every version of the application on the `main` branch is automatically packaged into docker images using the `main` tag. Every release is also published using the `latest` and respective version tags.
 
+
+## Deployment
+
+This repository contains all necessary files to deploy the web frontend to Google Cloud Firebase ([Stanford mHealth Platform](https://med.stanford.edu/mhealth.html)).
+
+
+### Deployment Configuration
+
+...
```

Comment:
Added some deployment SSO configuration information. Would be great if you can add some missing elements here about the secrets and the other general deployment information.